backend/cellguide/pipeline/computational_marker_genes/utils.py

```diff
@@ -177,14 +176,10 @@ def query_gene_info_for_gene_description(gene_id: str) -> str:
     The name of the gene if the query is successful, otherwise returns the gene ID.
     """
 
-    API_URL = os.getenv("API_URL")
-    if API_URL is None:
-        API_URL = "https://api.cellxgene.dev.single-cell.czi.technology"
-
     session = setup_retry_session()
-    response = session.get(f"{API_URL}/gene_info/v1/gene_info?gene={gene_id}")
+    response = session.get(f"http://rest.ensembl.org/lookup/id/{gene_id}", headers={"Content-Type": "application/json"})
```

Comment:
our gene info endpoint is very slow - we can use the ensembl API to get missing descriptions from the ensembl fixture